client/src/cordova/plugin/android/java/org/outline/OutlinePlugin.java

```diff
@@ -135,6 +137,10 @@ public void onServiceDisconnected(ComponentName className) {
   protected void pluginInitialize() {
     OutlineLogger.registerLogHandler(SentryErrorReporter.BREADCRUMB_LOG_HANDLER);
     Context context = getBaseContext();
+
+    final GoBackendConfig goConfig = Outline.getBackendConfig();
+    goConfig.setDataDir(context.getFilesDir().getAbsolutePath());
```

Comment:
I'm a little bit confused: are we setting the `dataDir` through `clientConfig` or here?